supporting_content/metrics/pricingquantity.md

```diff
@@ -0,0 +1,87 @@
+| Compute Usage        | 100                       | 1.0          | $100      |
+| Tax on Compute Usage | NULL                      | NULL         | $20       |
+| TOTAL                | 100                       | 1.0          | $120      |
+NOTE: Preferred option?
+
+## Example provider mappings
+
+Current column mappings found in available data sets:
+
+| Provider  | Data set                     | Column                   |
+|:----------|:-----------------------------|:-------------------------|
+| AWS       | Cost and Usage Report        | lineItem/UsageAmount (NOTE: how to handle reservation/TotalReservedUnits|
+| GCP       | BigQuery Billing             | usage.amount_in_pricing_units (NOTE: usage.amount provides QuantityInUsageUnit) |
+| Microsoft | Cost Details                 | Not available (NOTE: While Quantity is the closest, it provides QuantityInUsageUnit and not QuantityInPricingUnit) |
+| OCI       | Cost and Usage Report        | usage/billedQuantity  Note: usage/billedQuantity preferred over usage/billedQuantityOverage since the latter does not include the quantity covered by Universal Credits (commitment-based discounts) |
+
+## References and Resources
+
+### AWS
+
+- [Pricing details - AWS Cost and Usage Reports](https://docs.aws.amazon.com/cur/latest/userguide/pricing-columns.html)
+
+### GCP
+
+- [Structure of Detailed data export | Cloud Billing](https://cloud.google.com/billing/docs/how-to/export-data-bigquery-tables/detailed-usage)
+
+### Microsoft
+
+- [Understand usage details fields - Microsoft Cost Management](https://learn.microsoft.com/en-us/azure/cost-management-billing/automate/understand-usage-details-fields)
+
+### OCI
+
+- [Cost and Usage Reports Overview](https://docs.oracle.com/en-us/iaas/Content/Billing/Concepts/usagereportsoverview.htm)
+
+## Discussion / Scratch space
+
+ - Do we need to support negative values in the case of credits / refunds? (currently assumed yes)
+ - AWS issue “For size-flexible Reserved Instances, use the reservation/TotalReservedUnits column instead.”
+ - OCI seems to support separate billed quantities with an overage calculation… needs investigation.
+ - How will this apply for marketplace transactions…. What behavior do we expect for prices and quantities for marketplace purchases (and refunds)
+
+### ChargeType 'Tax'
+
+#### GCP
+ - Couldn’t find tax-related cost records in case of one small GCP account
+ - HS Data provided samples:
+     - usage.amount_in_pricing_units values: 1.0, 2.0, 4.0
+     - usage.pricing_unit: count
+
+#### AWS
+ - Tax-related records contain:
+     - lineItem/UsageAmount: 1
+     - pricing/unit: null
+
+#### Azure
+ - Tax-related record are not available in billing data
+
+##### OCI
+ - Tax-related record are not available in billing data
+
+## Example usage scenarios
+
+- See [Appendix: Pricing Support - UCs and Data samples Spreadsheet](../apendix/pricingsupport.md) for various UC scenarios
```

Comment:
Will this link work in the final version?  At the very least, "apendix" should be "appendix".

Reply:
Also, would this appendix show a comparison of usage unit and pricing unit?  I think such an example would be very useful to the layman practitioner.

Reply:
Thanks for pointing this out :) 
- Issues with links are resolved.
- The appendix and the Google spreadsheet require a bit more attention

Reply:
For now, fine to link to google docs.